src/actions/_rsaa.js

```diff
@@ -1,8 +1,61 @@
+import _ from 'lodash';
```

Comment:
Move and resculpture this file into `src/rsaa` directory. That holds:
 - index.js (for export definition)
 - headers.js (holds headers logic that you implemented),
 - meta.js (holds extendMetaWithResponse function), 
 - config.js (holds buildRSAAConfig from schemaConfig)